libraries/AP_EFI/AP_EFI_Serial_Hirth.cpp

```diff
@@ -369,23 +370,24 @@ void AP_EFI_Serial_Hirth::log_status(void)
     // @LoggerMessage: EFIS
     // @Description: Electronic Fuel Injection data - Hirth specific Status information
     // @Field: TimeUS: Time since system startup
-    // @Field: ETS1: Status of EGT1 excess temperature error
-    // @Field: ETS2: Status of EGT2 excess temperature error
-    // @Field: CTS1: Status of CHT1 excess temperature error
-    // @Field: CTS2: Status of CHT2 excess temperature error
-    // @Field: ETSS: Status of Engine temperature sensor
-    // @Field: ATSS: Status of Air temperature sensor
-    // @Field: APSS: Status of Air pressure sensor
+    // @Field: ES1: Status of EGT1 excess temperature error
+    // @Field: ES2: Status of EGT2 excess temperature error
+    // @Field: CS1: Status of CHT1 excess temperature error
+    // @Field: CS2: Status of CHT2 excess temperature error
+    // @Field: ETS: Status of Engine temperature sensor
+    // @Field: ATS: Status of Air temperature sensor
+    // @Field: APS: Status of Air pressure sensor
```

Comment:
Many of these are booleans.  You could save columns and logging bandwidth by logging them into a bitmask.

Many analysis tools do have reasonable support for bitmasks now.

Reply:
I have change the implementation and grouped all bool into 2 groups EET( Error Excess Temperature) FLAGS ( Sensor Status) I can also group them into one but thought to keep separate @peterbarker let me know your opinion on that. 
Also, will WriteStreaming be able to use FieldBitmaskEnum?

![image](https://github.com/user-attachments/assets/f5f7fe5b-03a7-4087-a0e0-69c5023f725a)
![image](https://github.com/user-attachments/assets/2aa44fe8-50bb-44bb-98aa-8ecb867bdcc6)